activity_calendar/templates/activity_calendar/slot_blocks/slot_block.html

```diff
@@ -0,0 +1,45 @@
+{% load static %}
+{% load calendar_tags %}
+
+<div class="card greenify activity-wide">
+    <div class="row no-gutters">
+        <div class="col-2 col-sm-1 activityImage">
+            <div class="heightener"></div>
+            <img src="{{slot.image_url}}" alt="" class="card-img">
+        </div>
+        <div class="col-10 col-sm-11 col-md-9">
+            <div class="card-body activityName">
+                <h5 class="card-title">
+                    {{ slot.title }}
+                </h5>
+                {% if slot.location %}
+                    <div class="row no-gutters">
+                        <small class="text-muted">
+                            Alternative Location: {{ slot.location }}
+                        </small>
+                    </div>
+                {% endif %}
+                <p class="card-text">{{ slot.description }}</p>
+
+                <div class="row no-gutters">
+                    <div class="card-text">
+                        <small class="text-muted">Participants:
```

Comment:
Why did you flip the order of Participant and the amount compared to the current version?

Reply:
Because I had this in my code before your code was imported into my code :P Seriously, I was working and every now and then I'd see a message in the chat addressing some Issue that I already addressed without knowing it XD

Reply:
I only added the list of participants in #63, the order of Participant and the amount did not change in that PR,

Reversing the order does solve the inflection issue that was present in the old version (1 Participants), but still I don't prefer this version, as I think that starting each line with "Participants: " together with the vertical looks a bit ugly as it doesn't form a correct sentence. What does Eric think?

Reply:
I agree with Henk. I'm personally not much of a fan of the vertical bar; it feels a little out of place.